src/main/java/mate/academy/internetshop/service/OrderService.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.internetshop.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import mate.academy.internetshop.model.Item;
+import mate.academy.internetshop.model.Order;
+import mate.academy.internetshop.model.User;
+
+/**
+ * @author Sergey Klunniy
+ */
+public interface OrderService {
+    Order completeOrder(List<Item> items, User user);
+
+    List<Order> getUserOrders(User user);
+
+    Order create(Order order);
+
+    Optional<Order> get(Long orderId);
```

Comment:
services should not return Optional (decided recently with Bogdan and group) so please, change to Order and so on